ThunderRequest/TSCRequestController.m

```diff
@@ -789,13 +776,13 @@ - (void)addCompletionHandler:(TSCRequestTransferCompletionHandler)handler progre
 	NSString *taskProgressIdentifierString = [NSString stringWithFormat:@"%lu-progress", (unsigned long)identifier];
 	
 	if ([self.completionHandlerDictionary objectForKey:taskIdentifierString]) {
-		NSLog(@"Error: Got multiple handlers for a single task identifier.  This should not happen.\n");
+        os_log_error(request_controller_log, "Error: Got multiple handlers for a single task identifier.  This should not happen.\n");
```

Comment:
Just noticed that all the requests are missing the log type. Everything should be set as Info, Debug, Error, Critical etc. Then that removes the need to have "Error:" at the beginning.

Reply:
I believe in objective-C these relate to separate functions: `os_log_error`, `os_log_debug` and `os_log_info` which we're using here and all look sensible to me!